app/src/main/java/fr/gaulupeau/apps/Poche/Poche.java

```diff
@@ -227,11 +212,13 @@ private void getSettings() {
         pocheUrl = settings.getString("pocheUrl", "http://");
         apiUsername = settings.getString("APIUsername", "");
         apiToken = settings.getString("APIToken", "");
+        nightmode = settings.getBoolean("Nightmode", false);
+
     }
 
     private void getDatabase() {
-	    if (database == null) {
-		    ArticlesSQLiteOpenHelper helper = new ArticlesSQLiteOpenHelper(this);
+        if (database == null || !database.isOpen()) {
```

Comment:
this was the reason for the crash that appeared if the app was closed (main activity) and reopened.
The db connection was not null but closed.